zebra-rpc/src/methods/tests/snapshot.rs

```diff
@@ -613,6 +613,12 @@ fn snapshot_rpc_getblockchaininfo(
                 // replace with:
                 "[Height]"
             }),
+            ".verificationprogress" => dynamic_redaction(|value, _path| {
+                // assert that the value looks like a valid verification progress here
+                assert!(value.as_f64().unwrap() < 1.0);
```

Comment:
```suggestion
                assert!(value.as_f64().unwrap() <= 1.0);
```

Q: If the chain is in sync, can this value be actually 1.0?

Reply:
Yep, applied as part of https://github.com/ZcashFoundation/zebra/pull/9215/commits/c62b61ef4e25ed4043db5a4ed82dc00963b0de74.